style.css

```diff
@@ -345,3 +357,221 @@ footer {
   border-radius: 10px;
   margin: 20px 0;
 }
+
+@media only screen and (min-width: 1024px) {
```

Comment:
- [ ] The desktop version looks great. However, you have designed the mobile version specifically for one screen size (376px), for this reason, it is not responsive.
![Screenshot from 2021-08-20 16-06-26](https://user-images.githubusercontent.com/65501535/130237718-1ddd3083-aa53-40b9-a55e-23f16473909e.png)